packages/components/pages/f-account-info/src/constants.js

```diff
@@ -0,0 +1,2 @@
+// eslint-disable-next-line import/prefer-default-export
+export const STOP_LOADING_SPINNER_EVENT = 'stop-spinner';
```

Comment:
Should this be placed in an events file / folder rather than constants so we have all events sitting in one area of the app.

Reply:
There will only be about x5 constant in the whole of the component and will be very similar to [this](https://github.com/justeat/fozzie-components/blob/master/packages/components/pages/f-contact-preferences/src/constants.js).

Reply:
A super small thing – I'd likely name this `SPINNER_EVENT_STOP_LOADING` or `EVENT_SPINNER_STOP_LOADING`, just as that way other potential events can have the same prefix `EVENT_blah_blah_blah` at the start of the constant name.